podman/domain/config.py

```diff
@@ -1,13 +1,16 @@
-"""Read containers.conf file."""
+"""Read containers.conf file and create Podman containers with network options."""
 
 import sys
 import urllib
 from pathlib import Path
 from typing import Dict, Optional
 import json
+import logging
 
 from podman.api import cached_property
 from podman.api.path_utils import get_xdg_config_home
+from podman import PodmanClient
```

Comment:
This line is giving a circular import. Please, remove it. There are ways to call the client such as `podman_client = self.client`